.github/workflows/linting.yml

```diff
@@ -0,0 +1,17 @@
+name: Linting Salt-Toaster
+
+on: [push, pull_request]
```

Comment:
IMO it does not make sense to run `pylint` twice for every PR (that's not from a fork)? How about we only run on `push` to `master`?

Reply:
Done